### PR TITLE
Container Grabber

### PR DESCRIPTION
**Pull request by celskeggs**
This should only be merged once #102 is merged. Feel free to code review it before then... but it includes all of the changes from #102 anyway, so start there.

This needs to be reviewed and merged by 3:00 on Tuesday. (And therefore, so does #102.)

Pinging @AidanSmith, @vincent-miller, and @amackworth for a code review.

### CODE REVIEW

**Comment by amackworth** [comment]
Could you give a brief description of what the container grabber looks like? (Just interested.)

**Comment by celskeggs** [comment]
Haven't seen it yet myself. I've also not been doing much robotics this break. I'll be in the lab shortly, however, and I can email you a photo.

**Comment by jacobbendicksen** [comment]
OK, now I’m interested. Can you send me the photo, too?

Thanks!!!

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeGrabContainer.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    public AutonomousModeGrabContainer() {
+        super("Grab Container");
+    }
+
+    @Override
+    protected void runAutonomous() throws InterruptedException, AutonomousModeOverException {
+        ContainerGrabber.containerGrabberSolenoid.set(true);
+        waitForTime(attachWaitingTime);
+        straightening.set(false);
+        DriveCode.strafe.set(strafeSpeed.get());
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeGrabContainer.java:20]: Are we allowed to strafe in auto now?

**Comment by amackworth** [comment]
Ah, thanks! What's the general idea behind it/what's it for?

(Also, other than my question and assuming the port number's correct, this looks good to me. :+1: I'll head over to review the other PR now.)

**Comment by celskeggs** [comment]
Actually, this PR is now cancelled. The can grabber failed. `lol` was correct. (Also, this was the second PR to do... not the first. But it doesn't matter now.)

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeGrabContainer.java (hunk 1 above)
Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeGrabContainer.java:20]: Not really, but this mode SPECIFICALLY required it.

**Pull request closed without merging**